bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v2/ImageImplTest.java

```diff
@@ -52,6 +55,15 @@ public void testExportedType() {
         assertEquals(ImageImpl.RESOURCE_TYPE, image.getExportedType());
     }
 
+    @Test
```

Comment:
Could you please move the `testImageWithNoSmartSize` and `testImageWithTwoOrMoreSmartSizes` methods back to their original position? They have no changes AFAICT